message_ix/tests/data/README.md

```diff
@@ -0,0 +1,16 @@
+# Nightly Tests
```

Comment:
There's an existing section describing these tests in [doc/contributing.rst](https://github.com/iiasa/message_ix/blob/main/doc/contributing.rst). Please either:
1. Expand that section, or
2. Create a new page within doc/contrib and link to it from contributing.rst, in the same way as the others.